Makefile

```diff
@@ -102,6 +102,9 @@ spanning_tree:
 vw:
 	cd vowpalwabbit; $(MAKE) -j $(NPROCS) things
 
+vwslim: vw
+	cd vowpalwabbitslim; $(MAKE) -j $(NPROCS) things
```

Comment:
>;  [](start = 20, length = 2)

change to && so make will only run if cd works

Reply:
Yeah I see your point. && will ensure the command before succeeds.  But we should do it in all places to be consistent.  Your call.

---
In reply to: [188103481](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188103481) [](ancestors = 188103481,188073071)